Cython/Utility/MemoryView.pyx

```diff
@@ -331,7 +331,8 @@ cdef class memoryview:
 
     cdef object obj
     cdef object _size
-    cdef object _array_interface
+    # This comes before acquisition_count so can't be removed without breaking ABI compatibility
+    cdef void* _unused
```

Comment:
I wonder if changing from refcounted `object` to `void*` is safe, considering different versions accessing it. Deallocation with `Py_CLEAR()` would still work, but I can't say if there are really no cases where `None` versus `NULL` makes a difference. Assuming that refounting `None` is as cheap as it gets, we can probably live with keeping the field as `object` just for safety. Renaming it to clearly mark it as unused is certainly ok, though.

Reply:
I don't think this should matter. The only place it's accessed from is the destructor, and each copy of the memoryview class will have its own version of that.

It's only really the attributes that are accessed through free utility code functions/macros  that matter, and this one's genuinely unused.